src/playerCommands.ts

```diff
@@ -165,59 +165,59 @@ export const commands = commandList({
 					outputSuccess(`Aoelog disabled.`);
 				}
 			},
-			tapped({x, y, output, outputFail, sender, admins, handleTaps, args}) {
-				function handleArea(p1: [number, number], p2: [number, number]){
+			tapped({ x, y, output, outputFail, sender, admins, handleTaps, args }) {
+				function handleArea(p1: [number, number], p2: [number, number]) {
 					const minX = Math.min(p1[0], p2[0]);
 					const maxX = Math.max(p1[0], p2[0]);
 					const minY = Math.min(p1[1], p2[1]);
 					const maxY = Math.max(p1[1], p2[1]);
 					let limitTiles = 0;
```

Comment:
Again you have added unnecessary `as any` casts. Why was this variable renamed to `amountLimit`? `amount` is fine, and maybe `limit` is a better name, but why `amountLimit`?